src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.hpp

```diff
@@ -50,16 +50,31 @@ class ShenandoahMmuTask;
  * MMU.
```

Comment:
I believe these have already been split out into a separate PR. Let's drop these from this PR by producing a dependent PR to avoid having to look at these diffs again.